main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
+%%
+
+%
+The Ag1000g variant data was functionally annotated using the SnpEff v4.1b software which allowed investigation of potential phenotype altering variants within Vgsc \cite{Cingolani2012}.
+%
+Non-synonymous \emph{Vgsc} variants were identified as all variants in AGAP004707, 2L:2358158-2431617, with a SnpEff annotation of “missense”  and an ALT allele frequency of >5\% in at least one of the nine mosquito populations, with the exceptions of the multi-allelic SNP 2L:2400071 G>A which is shown despite only being found in \emph{An. gambiae} from Cameroon at 0.4\% frequency, as the G>T variant at the same position which causes the same codon change (\texttt{M490I}), is found above 5\% frequency in Kenya.
+%
+\texttt{F1920S} is included for continuity with recent \emph{An. gambiae Vgsc} research \cite{Ag1000gConsortium2017}.
+%
+A minimum ALT allele frequency was employed to discriminate towards variants that may be undergoing selective sweeps and against less informative low frequency alleles.
+%%
+
+%
+For ease of comparison with previous work on \emph{Vgsc}, pan Insecta, \emph{An. gambiae} codon numbering is also reported in the equivalent \emph{Musca domestica} codon numbering, the species in which the gene was first discovered.
+%
+The \emph{M. domestica Vgsc} sequence (EMBL accession X96668 - \cite{Williamson1996}) was aligned with the \emph{An. gambiae} AGAP004707-RA sequence (AgamP4.4 gene-set), using the Mega v7 software package \cite{Kumar2016}.
+%
+A map of equivalent codon numbers between the two species can be download from the MalariaGEN website (@@include as supplementary data file?)- 
+\url{https://www.malariagen.net/sites/default/files/content/blogs/domestica_gambiae_map.txt}.
+%%
+
+%
+Haplotypes for each chromosome of each sample were estimated (phased) using using phase informative reads (PIRs) and SHAPEIT2 v2.r837 \cite{Delaneau2013}, see Ag1000g Consortium (\cite{Ag1000gConsortium2017}) supplementary text for more details.
+%
+The SHAPEIT2 algorithm is unable to phase multi-allelic positions, therefore the two multi-allelic non-synonymous SNPs within the \emph{Vgsc} gene (>5\% ALT frequency in at least one population), altering codons \texttt{V402} and \texttt{M420}, were phased onto the haplotypes using MVNcall v1.0 \cite{Menelaou2013}.
+%
+Conservative filtering had removed one of the three known insecticide resistance conferring kdr variants, \texttt{N1570Y} \cite{Jones2012}.
+%
+After manual inspection of the read alignment revealed that the SNP call could be confidently made, it was added back into the data set and then also phased onto the haplotypes using MVNcall (see \cite{Ag1000gConsortium2017}).
+%
+To evaluate the linkage disequilibrium (LD) of non-synonymous \emph{Vgsc} mutations with the two most widespread \emph{kdr} resistance mutations (\texttt{L995S/F}), the D1 statistic was calculated using haplotypes and the python library Scikit-allel \cite{Miles2016}.
```

Comment:
I would drop this sentence, don't need to say how D' was computed (and it's not actually in scikit-allel yet).